File: algorithms/ar-lsrh/python/ar-lsrh.py
def linear_search(arr, key):
    n = len(arr)
    for i in range(n):
        if arr[i] == key:
            return i
    return -1


arr = [1, 2, 5, 3, 7, 8, 89, 34, 23, 56, 99]
key = 1
ans = linear_search(arr, key)
if ans == -1:
    print("Key not found.")
else:
    print("Key " + str(key) + " found at index " + str(ans))

File: algorithms/ar-lsrh/c/ar-lsrh.c
#include<stdio.h>
int main()
{
    int n, arr[10000], key = 0, i, found = 0;
    printf("Enter Size of Array\n");
    scanf("%d", &n);
    printf("Enter Array elements\n");
    for (i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }
    printf("Enter Key to Search\n");
    scanf("%d", &key);
    for (i = 0; i < n; i++)
    {
        if (arr[i] == key)
        {
            found = 1;
            printf("found at position %d", i + 1);
        }
    }
    if (found == 0)
        printf("Not Found\n");

    return 0;
}
